src/main/java/hexgui/gui/GuiUtil.java

```diff
@@ -0,0 +1,314 @@
+    if (content == null || !content.isDataFlavorSupported(DataFlavor.stringFlavor)) return null;
+    try {
+      return (String) content.getTransferData(DataFlavor.stringFlavor);
+    } catch (UnsupportedFlavorException e) {
+      return null;
+    } catch (IOException e) {
+      return null;
+    }
+  }
+
+  /**
+   * Return a style sheet for message labels using HTML.
+   *
+   * @return A string with a HTML-head tag containing a style tag with formatting options or an
+   *     empty string.
+   */
+  public static String getMessageCss() {
+    if (Platform.isMac())
+      return "<head><style type=\"text/css\">"
+          + "b { font: 13pt \"Lucida Grande\" }"
+          + "p { font: 11pt \"Lucida Grande\"; margin-top: 8px }"
+          + "</style></head>";
+    else return "<head><style type=\"text/css\">" + "p { margin-top: 8px }" + "</style></head>";
+  }
+
+  /**
+   * Get size of default monspaced font. Can be used for setting the initial size of some GUI
+   * elements.
+   */
+  public static int getDefaultMonoFontSize() {
+    return MONOSPACED_FONT.getSize();
+  }
+
+  public static ImageIcon getIcon(String icon, String name) {
+    String resource = "hexgui/images/" + icon + ".png";
+    URL url = GuiUtil.class.getClassLoader().getResource(resource);
+    return new ImageIcon(url, name);
+  }
+
+  /**
+   * Manually break message into multiple lines for multi-line labels. Needed for multi-line
+   * messages in option panes, because pack() on JOptionPane does not compute the option pane size
+   * correctly, if a maximum width is set and the label text is automatically broken into multiple
+   * lines. The workaround with calling invalidate() and pack() a second time does not work either
+   * in this case. See also Sun Bug ID 4545951 (still in Linux JDK 1.5.0_04-b05 or Mac 1.4.2_12)
+   */
+  public static String insertLineBreaks(String message) {
+    final int MAX_CHAR_PER_LINE = 72;
+    int length = message.length();
+    if (length < MAX_CHAR_PER_LINE) return message;
+    StringBuilder buffer = new StringBuilder();
+    int startLine = 0;
+    int lastWhiteSpace = -1;
+    for (int pos = 0; pos < length; ++pos) {
+      char c = message.charAt(pos);
+      if (pos - startLine > 72) {
+        int endLine = (lastWhiteSpace > startLine ? lastWhiteSpace : pos);
+        if (buffer.length() > 0) buffer.append("<br>");
+        buffer.append(message.substring(startLine, endLine));
+        startLine = endLine;
+      }
+      if (Character.isWhitespace(c)) lastWhiteSpace = pos;
+    }
+    if (buffer.length() > 0) buffer.append("<br>");
+    buffer.append(message.substring(startLine));
+    return buffer.toString();
+  }
+
+  /**
+   * Call SwingUtilities.invokeAndWait. Ignores possible exceptions (apart from printing a warning
+   * to System.err
+   */
+  public static void invokeAndWait(Runnable runnable) {
+    try {
+      SwingUtilities.invokeAndWait(runnable);
+    } catch (InterruptedException e) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**: InterruptedException is ignored. This can delay thread shutdown and clear the thread’s interrupt status. Only code that implements a thread’s interruption policy can swallow an interruption request.

**Fix**: Rethrow the InterruptedException or reinterrupt the current thread using *Thread.currentThread().interrupt()* so that higher-level interrupt handlers can function correctly.
 If you are wrapping the InterruptedException inside a RuntimeException, call *Thread.currentThread().interrupt()* before throwing the RuntimeException.

Learn more about [interrupts](https://docs.oracle.com/javase/tutorial/essential/concurrency/interrupt.html) and [dealing with InterruptedException](https://www.ibm.com/developerworks/java/library/j-jtp05236/index.html?ca=drs-#N10187)